pysptk/sptk.pyx

```diff
@@ -2637,6 +2637,18 @@ def mglsadf(x, np.ndarray[np.float64_t, ndim=1, mode="c"] b not None,
 
     return _mglsadf(x, &b[0], order, alpha, stage, &delay[0])
 
+### Excitation ###
+def excite(np.ndarray[np.float64_t, ndim=1, mode = "c"] pitch, frame_period=100, interp_period=1, gaussian=False, seed=1):
```

Comment:
`hopsize` instead of `frame_period`? or rename `hopsize` to `frame_period` in `swipe` for naming consistency.